doc/api/path.md

```diff
@@ -431,6 +431,39 @@ added: v0.11.15
 The `path.posix` property provides access to POSIX specific implementations
 of the `path` methods.
 
+*Note*: Be careful when using the following functions directly on Windows.
+
+### path.posix.resolve([...path])
+If after processing all given `path` segments an absolute path has not yet
+been generated, `path.posix.resolve` will throw [`UNSUPPORTED_PLATFORM`] error.
+
+### path.posix.relative(from, to)
+If `from` and `to` are both relative path, `path.posix.relative` will use the
```

Comment:
Please include the `()` when referencing functions... e.g. `path.posix.relative()`